server/services/interventionsService.test.ts

```diff
@@ -106,6 +106,38 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
         ])
       })
     })
+
+    describe('for a referral that has had a service user selected', () => {
+      beforeEach(async () => {
+        await provider.addInteraction({
+          state:
+            'There is an existing draft referral with ID of 1219a064-709b-4b6c-a11e-10b8cb3966f6, and it has had a service user selected',
+          uponReceiving: 'a request for that referral',
+          withRequest: {
+            method: 'GET',
+            path: '/draft-referral/1219a064-709b-4b6c-a11e-10b8cb3966f6',
+            headers: { Accept: 'application/json', Authorization: 'Bearer token' },
+          },
+          willRespondWith: {
+            status: 200,
+            body: Matchers.like({
+              id: '1219a064-709b-4b6c-a11e-10b8cb3966f6',
+              serviceUser: {
```

Comment:
do we want this in the referral - or as an id referancing another entity as is the case with complexity level, service category etc?

Reply:
I don't know. I just did the simplest thing possible that would get me a first name I could work with. I guess I thought of the service user data as just being part of the "referral form", which is the data contained in the base referral object, but that's a bit of a flimsy reasoning since complexity level and service category are part of the referral form too.

Do you think we should try and figure this out in this PR or leave it until we come to implement service user details properly? I'm conscious that by merging this PR then we'd add a burden for it to be implemented on the backend, but at the same time I'm keen for this frontend functionality not to get blocked by a conversation about API design.

Reply:
makes sense. we can leave like this for now and see how we get on. 👍🏼